doc/topics/releases/3005.rst

```diff
@@ -1,7 +1,135 @@
++--------------+---------------------+---------------------+
+| Windows      |         yes         |        yes          |
++--------------+---------------------+---------------------+
+
+
+Repo paths
+----------
+
++----------+-----------------------------------------------+-----------------------------------------+
+|     OS   | Tiamat Path                                   | Non-Tiamat Path                         |
++==========+===============================================+=========================================+
+| RHEL     | https://repo.saltproject.io/salt/py3/redhat/  | https://repo.saltproject.io/py3/redhat/ |
++----------+-----------------------------------------------+-----------------------------------------+
+| Ubuntu   | https://repo.saltproject.io/salt/py3/ubuntu/  | https://repo.saltproject.io/py3/ubuntu/ |
++----------+-----------------------------------------------+-----------------------------------------+
+| Debian   | https://repo.saltproject.io/salt/py3/debian/  | https://repo.saltproject.io/py3/debian/ |
++----------+-----------------------------------------------+-----------------------------------------+
+| Raspbian | https://repo.saltproject.io/salt/py3/debian/  | https://repo.saltproject.io/py3/debian/ |
++----------+-----------------------------------------------+-----------------------------------------+
+| Fedora   | Hosted on Fedora Repos                        | Hosted on Fedora Repos                  |
++----------+-----------------------------------------------+-----------------------------------------+
+| MacOS    | https://repo.saltproject.io/salt/py3/osx/     | https://repo.saltproject.io/osx/        |
++----------+-----------------------------------------------+-----------------------------------------+
+| Windows  | https://repo.saltproject.io/salt/py3/windows/ |  https://repo.saltproject.io/windows/   |
++----------+-----------------------------------------------+-----------------------------------------+
+
+Note that the Tiamat paths above will not be available until the day of the Phosphorus release.
+
+
+How do I migrate to the Tiamat packages?
+----------------------------------------
+The migration path from the non-Tiamat packages to the Tiamat packages will include:
+
+* Repo File: You need to update your repo file to point to the new repo paths for your platform. After the repo
+  file is updated, upgrade your Salt packages.
+* Pip packages: You need to ensure any 3rd party pip packages are installed in the correct Tiamat path.
+  This can be accomplished in two ways:
+
+  * ``salt pip install <package name>``
+  * Using the ``pip.installed`` Salt state.
+
+To install python packages into the system python environment, user's must now provide the ``pip_bin`` or ``bin_env`` to the pip state module.
+
+For example:
+
+.. code-block:: yaml
+
+    lib-foo:
+      pip.installed:
+        - pip_bin: /usr/bin/pip3
+    lib-bar:
+      pip.installed:
+        - bin_env: /usr/bin/python3
```

Comment:
**Suggestion:** Rather than duplicating install instructions in both the release notes and the install guide, we should probably just link to the install guide. Do you need me to get those URLs set up now?